pkg/api/routers/metric.go

```diff
@@ -22,6 +22,16 @@ import (
 
 func init() {
 
+	// add router for file ops api
+	filens :=
+		beego.NewNamespace("/"+constants.APIVersion,
```

Comment:
Please append it in the existing namespace.

Reply:
done